integrations/opensearch/src/haystack_integrations/components/retrievers/opensearch/embedding_retriever.py

```diff
@@ -86,10 +156,13 @@ def run(self, query_embedding: List[float], filters: Optional[Dict[str, Any]] =
             filters = self._filters
         if top_k is None:
             top_k = self._top_k
+        if custom_query is None:
+            custom_query = self._custom_query
 
         docs = self._document_store._embedding_retrieval(
             query_embedding=query_embedding,
             filters=filters,
             top_k=top_k,
+            custom_query=custom_query,
```

Comment:
same as above, why not pass directly the parameter attached to the `self`?

Reply:
see above